src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,78 @@
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const totalPages = Math.ceil(getNumbers(1, total).length / perPage);
+  const totalPagesArr = getNumbers(1, totalPages);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn(
+          'page-item',
+          { disabled: currentPage === 1 },
+          { active: currentPage === 1 },
+        )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
```

Comment:
The `onClick` handler should prevent the default action to ensure the page does not jump to the top when the link is clicked. You can do this by accepting the event as a parameter and calling `event.preventDefault()`.